src/components/LoginStatus.test.tsx

```diff
@@ -40,6 +65,16 @@ describe("LoginStatus", () => {
         loadingState={LoadingState.PENDING}
       />
     );
-    expect(component).toMatchSnapshot();
+    expect(component).toMatchInlineSnapshot(`
+      <div
+        className="login-status"

```

Comment:
Hmmm... I would expect the modifier at this level, or at least these to class names should be together somewhere (`className="login-status login-status--loading"`)